core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -225,6 +225,7 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
   private var _jars: Seq[String] = _
   private var _files: Seq[String] = _
   private var _shutdownHookRef: AnyRef = _
+  private var _logUrls: Option[Predef.Map[String, String]] = None
```

Comment:
Unfortunately that cause scala compiler to fail to build, since the Predef.Map is the one that is taken by the `SparkListenerExecutorAdded` and we already import `mutable.Map` in this class, so we need to specify that this one is a `Predef.Map`